src/test/java/hw9/BoardTest.java

```diff
@@ -0,0 +1,112 @@
+package hw9;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import entity.BoardEntity;
+import entity.ListEntity;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.testng.annotations.Test;
+import utils.Endpoints;
+
+
+public class BoardTest extends BasicTest {
+
+    static String boardName = RandomStringUtils.random(40, true, true);
+
+    static String listName = RandomStringUtils.random(40, true, true);
+
+    @Test
+    public static void createBoard() {
+        board = given().spec(baseRequestSpecification)
+                .when()
+                .queryParam("name", boardName)
+                .basePath(Endpoints.Board_Url)
+                .post()
+                .then()
+                .body("name", equalTo(boardName))
+                .spec(responseSpecification).log().all()
+                .extract().body().as(BoardEntity.class);
+    }
+
+    @Test
+    public void getInfoAboutBoard() {
+        if (board == null) {
+            createBoard();
+        }
```

Comment:
Two points:
1) Really bad practice - conditional testing. Find a way to be rest assured you always have created board to cover this scenario. 
2) Never use tests inside tests. It feels like you need to have additional layer, like service object for that.

Reply:
Task: go through all test cases and remove conditions